### PR TITLE
Describe updated configurations of IR and Node in env files

### PR DESCRIPTION
**Pull request by cthulhu-rider**
SN config was cleaned up during refactoring. Some changes also affected IR config (profiler, metrics).

### CODE REVIEW

File: services/ir/.ir.env [hunk 1]
```diff
@@ -1,6 +1,20 @@
 NEOFS_IR_LOGGER_LEVEL=debug
-NEOFS_IR_METRICS_ENABLED=true
-NEOFS_IR_PPROF_ENABLED=true
+
+# Profiler section
+## Server address. Empty value disables profiler
+## Default: <empty> (invalid)
```
Comment (alexvanin) [inline services/ir/.ir.env:5]:
```suggestion
## Default: <empty>
```
For metrics too.

File: services/ir/.ir.env [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+#NEOFS_IR_PROFILER_ADDRESS=:6060
```
Comment (alexvanin) [inline services/ir/.ir.env:6]: Profiler and metrics were enabled, let's keep it this way.

File: services/storage/.storage.env [hunk 3]
```diff
@@ -1,18 +1,82 @@
+# Logger section
+## Minimum enabled logging level
+## Default: info
 NEOFS_LOGGER_LEVEL=debug
 
+# Profiler section
+## Server address. Empty value disables profiler
+## Default: <empty> (invalid)
+#NEOFS_PROFILER_ADDRESS=:6060
```
Comment (alexvanin) [inline services/storage/.storage.env:9]: See comments about profiler and metrics from `.ir.env`

File: services/ir/.ir.env (hunk 1 above)
Comment (cthulhu-rider) [reply services/ir/.ir.env:5]: Means that empty value is invalid

Comment (alexvanin) [reply services/ir/.ir.env:5]: The line above says that empty value disables profiler, so I support is it valid, isn't it?

Comment (cthulhu-rider) [reply services/ir/.ir.env:5]: Ok, it's valid in program, but invalid way to serve profiler. I'll remove `invalid`

Comment (alexvanin) [reply services/ir/.ir.env:5]: Maybe `## Default: <empty> (disabled)` will be better.

File: services/ir/.ir.env [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+## Default: <empty> (disabled)
+NEOFS_IR_PROFILER_ADDRESS=:6060
+## Timeout to shut down the HTTP server
+## Default: 30s
+#NEOFS_IR_PROFILER_SHUTDOWN_TIMEOUT=1m
+
+# Application metrics section
+## Server address. Empty value disables metrics gathering
+## Default: <empty> (disabled)
+NEOFS_IR_METRICS_ADDRESS=:6060
```
Comment (alexvanin) [inline services/ir/.ir.env:14]: Use `:9090` for metrics in IR and SN.

**Pull request merged**